src/common/Set/style.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+
+export const SectionInfo = styled.div`
+  margin-bottom: 0.75em;
+`;
+export const SectionTitle = styled.div`
+  font-family: "Roboto Mono", monospace;
+  font-size: 1.5em;
+  font-weight: bolder;
+`;
+export const SectionDescription = styled.div`
+  font-family: "Roboto Mono", monospace;
+  font-weight: 100;
+`;
+export const CardsContainer = styled.div`
+  display: grid;
+  grid-gap: 0.5em;
+  grid-template-columns: auto auto auto auto;
+  @media (max-width: 1200px) {
+    display: flex;
+    overflow: scroll;
+    &::-webkit-scrollbar {
+      display: none;
+      -webkit-appearance: none;
+    }
+  }
```

Comment:
```suggestion
  @media (max-width: 1200px) {
    display: flex;
    overflow: auto;
    & > div {
      width: 80vw;
      max-width: 350px;
    }
    &::-webkit-scrollbar {
      display: none;
      -webkit-appearance: none;
    }
  }
```

This is a change to make the Cards look good in mobile views (horizontal scrolling). You also have to use the suggested change in the card styles.tsx to see the changes correctly.